Sigrun/src/components/AppHeader.tsx

```diff
@@ -109,7 +109,7 @@ export function AppHeader({ dark, toggleColorScheme, toggleDimmed, saveLang }: A
   const [onlineError, setOnlineError] = useState<string | null>(null);
   const api = useApi();
   const tryAddOnline = () => {
-    if (!onlineLink.match(/^https?:\/\/[^\/]+\/\d\/\?log=\d+gm-\d+-\d+-[0-9a-f]+/i)) {
+    if (!onlineLink.match(/^(https|http)+:\/\/[^\/]+\/\d\/\?log=\d+gm-[0-9a-f]+-\d+-[0-9a-f]+/i)) {
```

Comment:
About the first change in line: I guess `+` is not required here. Actually without `+` these are equivalent expressions, why change it?